runtime/jvmti/jvmtiClass.c

```diff
@@ -2674,27 +2674,27 @@ jvmtiGetConstantPool_addMethodHandle(jvmtiGcp_translation *translation, UDATA cp
 	case MH_REF_PUTSTATIC:
 		fieldOrMethodCpType = CFR_CONSTANT_Fieldref;
 		break;
-
 	case MH_REF_INVOKEVIRTUAL:
 	case MH_REF_INVOKESTATIC:
 	case MH_REF_INVOKESPECIAL:
 	case MH_REF_NEWINVOKESPECIAL:
 		fieldOrMethodCpType = CFR_CONSTANT_Methodref;
 		break;
-
 	case MH_REF_INVOKEINTERFACE:
 		fieldOrMethodCpType = CFR_CONSTANT_InterfaceMethodref;
 		break;
 	default:
-		Assert_JVMTI_true(0);
+		Assert_JVMTI_unreachable();
 		return JVMTI_ERROR_INTERNAL;
 	}
 
 	/* Ensure the field/method ref is also added */
-	return jvmtiGetConstantPool_addReference(translation, ref->methodOrFieldRefIndex,
-						fieldOrMethodCpType,
-						(J9ROMFieldRef *) &translation->romConstantPool[ref->methodOrFieldRefIndex],
-						sunCpIndex);
+	return jvmtiGetConstantPool_addReference(
+				translation,
+				ref->methodOrFieldRefIndex,
+				fieldOrMethodCpType,
+				(J9ROMFieldRef *) &translation->romConstantPool[ref->methodOrFieldRefIndex],
```

Comment:
Please be consistent: you remove the space after the cast on line 2657, but not here; change both or neither.

Reply:
Updated.